docs/content/fantomas-setting-icon.js

```diff
@@ -0,0 +1,28 @@
+import { html } from 'https://cdn.skypack.dev/lit';
+import { component } from 'https://cdn.skypack.dev/haunted';
+
+
+function FantomasSettingIcon({ tooltip, type }) {
+    let settingType = 
```

Comment:
Use `const` instead of `let`

Reply:
Going from the old expression to a statement using switch case, I couldn't really come up with a way of using a `const` .
I did change the below 'lets' to const. Hope that works.